lib/ui/src/modules/ui/libs/hierarchy.js

```diff
@@ -1,9 +1,36 @@
+function findMatches(matches, type, value) {
+  if (!matches) {
+    return null;
+  }
+
+  const matchForNamespace = matches
+    .filter(match => match.key === type)
+    .find(match => match.value === value);
+
+  if (!matchForNamespace) {
+    return null;
+  }
+
+  return matchForNamespace.indices.filter(([start, end]) => end - start > 0);
```

Comment:
It just looks weird with one letter highlighting
<img src="https://user-images.githubusercontent.com/7867954/29238688-f45f97be-7f42-11e7-8f94-c7fddde70a3b.png" height="400">